omod/src/main/webapp/resources/scripts/fragments/patientdashboard/encountertemplate/defaultEncounterTemplate.js

```diff
@@ -14,7 +14,7 @@ $(function() {
 		showDeleteEncounterDialog();
 	});
 
-    $(document).on('click', '.editEncounter', function(event) {
+    $(document).on('click', '.editEncounter, .viewEncounter', function(event) {
```

Comment:
@dkayiwa Because when viewing the encounter you are not editing it... Sematically its wrong, as it does not convey the proper actions for the link. Let me clean up via uicommons and update